api/src/controllers/record-utils.js

```diff
@@ -210,28 +195,22 @@ const createByForm = (data, { locale }={}) => {
 };
 
 const createRecordByJSON = data => {
-  const required = ['from', 'form'],
-        optional = ['reported_date', 'locale'];
```

Comment:
optional fields don't really make sense, we will just not use what we don't know about